openmetadata-spec/src/main/resources/json/schema/entity/data/dataModel.json

```diff
@@ -0,0 +1,130 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/data/dataModel.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "DataModel",
+  "$comment": "@om-entity-type",
+  "description": "Data Model entity definition. Data models are the schemas used to build dashboards, charts, or other data models.",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.entity.data.DataModel",
+  "javaInterfaces": ["org.openmetadata.schema.EntityInterface"],
```

Comment:
There are multiple DataModels today. Should we call this type as DashboardDataModel? This will avoid conflict of multiple java classes named as DataModel.

Reply:
Sorry, maybe I'm overstepping, but a data model is not only for building dashboards. I would at least including "Reports and Dashboards". In the "Power BI" world a datamodel is used in several different places, but mainly Interative- and Static reports and dashboard .

Reply:
I think it is fine to call it `DashboardDataModel`, I didn't like the idea of repeated class names. @Daandamhuis when we refer here to `DashboardDataModel` we said that the DataModel is part of a `DashboardService`. This is a first iteration where we want to extract the `DataModels` and create a lineage from `Data Source` > `Data Model` > `Dashboard`. We can keep iterating later to make it more precise in case the `Data Model` can be associated to a `Dashboard` or a `Report`, etc.